meson.build

```diff
@@ -185,6 +185,10 @@ if python_command == '' or python_command == 'auto'
 endif
 cdata.set('PYTHON_COMMAND', python_command)
 
+if host_machine.system() == 'windows'
```

Comment:
This is [fine](https://mesonbuild.com/Cross-compilation.html), but we probably should stop using `target_machine` everywhere else…